src/Zipper.elm

```diff
@@ -952,6 +1033,9 @@ changeToAlpha z =
                     Refl t ->
                         Tableau tableau.node (Alpha t)
 
+                    Leibnitz t ->
+                        Tableau tableau.node (Alpha t)
+
                     _ ->
                         tableau
```

Comment:
```elm
\tableau -> Tableau tableau.node (Alpha (subTableauToKeep tableau))  
```
mozno nechat ten case  Alpha vs ostatne ako optimalizaciu ked netreba menit ... ;)

Podobne sa asi da v ostatnych  (akurat pri niektorych asi treba naozaj ten case aby sa zachovali subst a pod... ale so spravnym `maybeSubstitution` sa to tiez da dost pojednotit asi)